doc/how_to_build_linux.md

```diff
@@ -40,6 +40,7 @@ $ sudo apt-get install -y libopencv-dev
 Notes:
 * It's possible we also need `libgsl2` (or maybe `libopenblas-dev`)
 * For Qt, MyPaint and OpenCV, you can alternatively build and install from source.
+* On Ubuntu qt59serialport can be found under the name of libqt5serialport5 and libqt5serialport5-dev
```

Comment:
I apologize for having taken so long to look at this.

I build on Ubuntu 16.04 and use qt59seriealport.  What version of Ubuntu are you using and what version of QT are you building with.  What's the repo are you pulling QT from? 

Perhaps a little more clarification is needed on the line you are adding, maybe suggesting if qt59serialport cannot be found to look for the ones you mentioned?